public/lessons/client_side_coding/media.md

```diff
@@ -1,32 +1,54 @@
+
+You can include an image in your HTML documents by using an `<img>` tag with a `src` attribute pointing to either a URL where the image is hosted, or a relative file path to the location of the image in your file structure.
+
 # Image Types
 
-There are many different types of images, however we are going to focus on the three most common types of images you'll find around the web
+There are many different types of images (svg, png, ptg, psd, krd, ai, gif, jpg, and the list goes on), however we are going to focus on the three most common types of images you'll find around the web; JPGs (or JPEGs), PNGs, and SVGs.
 
-### jpeg/jpg
+# jpeg/jpg
 
 Your most basic image type. These images tend to have very small file sizes, but they also don't have quite as many features as other image types.
 
 These are pixel based images so they won't scale very well beyond their original size. They also don't support transparency, and so will block anything behind them in a rectangle with the dimensions of the image.
 
-JPGs and JPEGs are the same type of image, however when embedding them in your page you will need to make sure you are using the correct file extension otherwise your site won't be able to locate the image
+JPGs and JPEGs are the same type of image, however when embedding them in your page you will need to make sure you are using the correct file extension otherwise your site won't be able to locate the image. Also **case matters!**
 
-### png
+# png
 
-PNGs, like JPGs, are a pixel based image type, and as such do not scale well beyond their original size. However, unlike JPGs, PNGs suppport transparency, and as such can appear to be irregular shapes.
+PNGs, like JPGs, are a pixel based image type, and as such do not scale well beyond their original size. However, unlike JPGs, PNGs support transparency, and as such can appear to be irregular shapes.
 
-Despite how they appeare they are still rectangular and as such will prevent you from interacting with elements that are behind them even though you can see through them.
+Despite how they appear they are still rectangular and as such will prevent you from interacting with elements that are behind them even though you can see through them.
 
-### svg
+# svg
 
 SVG stands for **standard vector graphic** and unlike other types of images they are not made out of pixels.  They are instead collections of instructoins that tell the computer how to draw a specific shape.
 
-Since they are not blocks of pixels SVGs can be scaled up indefinately with no detrinmental effects on quality. They also are **exactly** the shape they appear to be and as such will only block what they appear to cover.
+Since they are not blocks of pixels SVGs can be scaled up indefinately with no detrinmental effects on quality, and they have very small file sizes compared to the other two image types covered above. They also are **exactly** the shape they appear to be and as such will only block what they appear to cover.
```

Comment:
Since they are not blocks of pixels`,` SVGs can be scaled up `indefinitely` with no `detrimental`